bear-requirements.txt

```diff
@@ -4,11 +4,10 @@ apertium-lint~=0.29
 autoflake~=0.7
 autopep8~=1.2
 bandit~=1.2
-click==6.6
 cmakelint~=1.3
 cppclean~=0.12.0
 cpplint~=1.3
-dennis~=0.8
+dennis~=0.9
```

Comment:
I think `~=` means `not equal to` and as per issue you have to update it to `version =0.9`, I think have to make it to `dennis==0.9` instead `dennis~=0.9`

Reply:
unack 996b0b2570a6c498fda2a30aa05beca5c8b19c44

Reply:
No, we avoid `==` if possible.

Reply:
ack 996b0b2